### ISSUE TITLE
Difficulty adding block at bottom of content

### ISSUE THREAD

**Issue by smerriman** [WordPress/gutenberg#64889]
### Description

Up to WP 6.5, it was possible to insert a block at the bottom of the editor by clicking in the space there. In WordPress 6.6, this is no longer possible.

This appears to have been instigated by https://github.com/WordPress/gutenberg/pull/60697 . 

The removal of the ability to insert blocks has caused other bugs; for example, it broke spacing to metaboxes ( https://github.com/WordPress/gutenberg/issues/63882 ). An accessibility concern was also raised that it was impossible for people to insert a block at the bottom with keyboard only ( https://github.com/WordPress/gutenberg/issues/61391 ). However, the (past + future) fixes for those two issues do not help regular users.

It appears the 'intended' solution is now to select the last block, and then press enter. This makes sense if the last block was a plain paragraph, as you see the blinking cursor and it is completely logical to press enter to move to another line. If, the last block is something else - for example, column content - it is completely illogical to regular site editors that you have to click a paragraph within the block, shift the focus to the wrapping columns block, then press enter. In fact, in the majority of (all?) editors outside of WordPress, selecting something in the editor then pressing enter would actually overwrite / delete that content.

Myself and others are downgrading clients to WP 6.5 as the current behavior is too confusing.

### Step-by-step reproduction instructions

1. Create a page with some blocks.
2. Click at the bottom of the editor.

### Environment info

WP 6.6, no Gutenberg plugin.

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by Mamaduka** [comment]
Probably a duplicate of #63601.

cc @talldan

**Comment by talldan** [comment]
I think it's a little different. My issue is more related to focusing/moving the caret into an existing last block, whereas this one is about adding a new block to the end of content.

**Comment by talldan** [comment]
cc @jasmussen @richtabor Seems like users are having issues inserting blocks at the bottom of the post editor after https://github.com/WordPress/gutenberg/pull/60697.

There were also a few comments in #63882.

**Comment by jasmussen** [comment]
CC: @WordPress/gutenberg-design

**Comment by paaljoachim** [comment]
Thank you Joen for bringing attention to this issue!

Test site.
WP version 6.6.1
Gutenberg 19.1.0
**EDIT. I happen to have Autoptimize plugin activated as well. Which seems to create the following behavior.**
I made an issue on the plugin support forum.
https://wordpress.org/support/topic/not-able-to-click-below-block-to-create-a-new-block-in-gutenberg/#new-topic-0

----

Yes. I have noticed this behavior for a while as well on a few sites.

I need only add one block. When I try to click below the one block I am not able to add another block by clicking below it. I have to use enter/return on my keyboard to be able to add another block.

https://github.com/user-attachments/assets/c54e1426-9f08-4828-bd5d-ff791ba11d8b

Use case.
I added paragraph block and a spacer block below it. Because I am not able to click below the spacer block I have had to click into the paragraph block right above the spacer block. Click enter, write something and then move the new paragraph block below the spacer block. 

https://github.com/user-attachments/assets/940d5702-1178-4c5f-b742-918db28bedd2


I really look forward to getting this fixed.

**Comment by paaljoachim** [comment]
Autoptimize plugin support mentioned to **disable** through Settings -> Autoptimize.
Enable configuration per post/ page? 
Add a "metabox" to the post/ page edit screen allowing different optimizations to be turned off on a per post/ page level?

---> Disabling to add a metabox also removed the issue of not being able to click below the current block to add a new block. This sounds like adding a metabox might cause this issue.

**Comment by paaljoachim** [comment]
I just tested with Yoast SEO plugin as it adds a metabox into the sidebar Page settings.
The problem is also seen here. I am not able to click below a current block to add a new block.

**Comment by paaljoachim** [comment]
I went ahead and reopened this issue:
**Post Edit Screen Restricted by Metaboxes**
https://github.com/WordPress/gutenberg/issues/63882

EDIT: My mistake the issue should not have been reopened. It was closed again.

**Comment by talldan** [comment]
It looks like there was a recent regression with the click below to add a block after #64639 in the iframed editor. The PR #64992 fixes it in my testing, so hopefully that can be finalized.

For some background this feature never worked in the non-iframed editor, so now it becomes clear why there were complaints after the trailing appender was removed. #64639 & #64992 will mean that iframed and non-iframed editor work the same way, so that shouldn't be an issue, and I think from 6.7 the iframed editor will hopefully become more prevalent.

**Comment by t-hamano** [comment]
Would #64639 and #64992 have solved this problem?

**Comment by stokesman** [comment]
It does appear this is resolved. I tested WP 6.6.2 in both iframed and non-iframed contexts and with or without meta boxes present. I did notice one issue with non-iframed contexts but I found an issue for that has been opened #65849. So I’ll close this.
